emails/utils.py

```diff
@@ -188,27 +194,36 @@ def get_welcome_email(request: HttpRequest, user: User, format: str) -> str:
     )
 
 
+MessageBodyContent = dict[Literal["Charset", "Data"], str]
+MessageBody = dict[Literal["Text", "Html"], MessageBodyContent]
+AttachmentPair = tuple[str, IOBase]
+MailJSON = dict[str, Any]
```

Comment:
suggestion (non-blocking): maybe someday we can start making modules like `emails/types.py` to hold our types.